turbo.json

```diff
@@ -20,17 +20,21 @@
       "outputs": ["dist/**"],
       "dependsOn": ["^build", "compile"]
```

Comment:
```suggestion
      "dependsOn": ["^build", "^build:docker", "compile"]
```

Reply:
We need this due to the pipeline being:

1. testsuite: `test`
2. testcontainer: `build`
3. node: `build:docker`

Hence `test` dependsOn `build ` which dependsOn `build:docker`

```
test
  build
    build:docker
```